test/source/TestAlgorithm.cpp

```diff
@@ -116,6 +118,57 @@ struct TestObjectNegate : public eastl::unary_function<TestObject, TestObject>
 		{ return TestObject(-a.mX); }
 };
 
+struct MissingMoveConstructor
```

Comment:
Ok. Moved them over. The structs I added are more like "negative" examples than the "positive" examples in that file.